doc/classes/CanvasItem.xml

```diff
@@ -126,7 +126,7 @@
 			<argument index="1" name="color" type="Color" />
 			<argument index="2" name="width" type="float" default="1.0" />
 			<description>
-				Draws multiple, parallel lines with a uniform [code]color[/code].
+				Draws multiple disconnected lines with a uniform [code]color[/code]. When drawing large amounts of lines, this is faster than using individual [method draw_line] calls. To draw interconnected lines, use [method draw_polyline] instead.
```

Comment:
I think multiline is meant to pass line *segments* into `points`, if this is the case then I think `points` parameter could also be renamed to `segments` or so, otherwise this will draw disconnected lines indeed.

Reply:
In 3.x it uses the GL_LINES OpenGL primitive, which is indeed disconnected it looks like:

> GL_LINES: Vertices 0 and 1 are considered a line. Vertices 2 and 3 are considered a line. And so on. If the user specifies a non-even number of vertices, then the extra vertex is ignored.

https://www.khronos.org/opengl/wiki/Primitive

`width` and `anti-aliased` parameters are unused in 3.x. I'll try and have a look in 4.x and see if it is similar.

My guess is this was designed for e.g. drawing grids, but I don't know how often it is used, I can see it being used in animation track editor and audio stream editor. Compare to `add_polyline` which is for continuous rather than disjointed lines.

BTW on 3.x at least, this won't be that much faster than individual lines, as they are now batched. But this may be quite a bit faster for grids etc on 4.x as there is no batching.